### PR TITLE
GP - Correct issue with IRS 1099 data migration

### LINKED ISSUE #26487: [INTERNAL] GP Updates
### Describe the issue

Issue to link GP update PRs to.

### Expected behavior

GP Updates merged.

### Steps to reproduce

N/A

### Additional context

_No response_

### I will provide a fix for a bug

- [X] I will provide a fix for a bug

### PR DESCRIPTION
**Pull request by jaymckinney-enavate**
This change corrects an issue when migrating GP Vendor 1099 data on version v25.

Changes:

- Always check to see if the new IRS Forms feature is enabled.
- Initialize the original/old IRS 1099 Forms feature if required.

Fixes #26487
Fixes [AB#560225](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/560225)

### CODE REVIEW

**Review by nikolakukrika — DISMISSED** [review]

**Comment by JesperSchulz** [comment]
Internal build error:
C:\Users\LocalUser\AppData\Local\Temp\1\0jwu5b40.ans\src\Codeunits\GPCloudMigrationUS.Codeunit.al(81,34): error AL0185: Codeunit 'Feature Management Facade' is missing

**Comment by jaymckinney-enavate** [comment]
How is that possible?
The "Feature Management Facade" Codeunit is part of the System Application.


________________________________

File: Apps/US/HybridGP_US/app/src/Codeunits/GPCloudMigrationUS.Codeunit.al [hunk 1]
```diff
@@ -3,9 +3,11 @@ namespace Microsoft.DataMigration.GP;
 using System.Integration;
 #if not CLEAN25
 using System.Environment.Configuration;
+using Microsoft.Finance.VAT.Reporting;
 #endif
 using Microsoft.Purchases.Vendor;
```
Comment (JesperSchulz) [inline Apps/US/HybridGP_US/app/src/Codeunits/GPCloudMigrationUS.Codeunit.al:8]:
```suggestion
using System.Integration;
using System.Environment.Configuration;
#if not CLEAN25
using Microsoft.Finance.VAT.Reporting;
#endif
using Microsoft.Purchases.Vendor;
```

File: Apps/US/HybridGP_US/app/src/Codeunits/GPCloudMigrationUS.Codeunit.al [hunk 2]
```diff
@@ -73,17 +78,10 @@ codeunit 42004 "GP Cloud Migration US"
...
     var
-#if not CLEAN25
         FeatureManagementFacade: Codeunit "Feature Management Facade";
-#endif
         IsEnabled: Boolean;
     begin
-        IsEnabled := true;
-
-#if not CLEAN25
         IsEnabled := FeatureManagementFacade.IsEnabled(IRSFormFeatureKeyIdTok);
```
Comment (JesperSchulz) [inline Apps/US/HybridGP_US/app/src/Codeunits/GPCloudMigrationUS.Codeunit.al:84]: The IRSFormFeatureKeyIdTok is removed if CLEAN25. Either also remove the call, or leave the variable declaration unconditional.

Comment (jaymckinney-enavate) [reply Apps/US/HybridGP_US/app/src/Codeunits/GPCloudMigrationUS.Codeunit.al:82]: Leaving the variable and removed the condition.

**Review by JesperSchulz — APPROVED** [review]

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 27997.

**Pull request merged**